scripts/ceph_poll.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+while true; do
+    kubectl -n rook-ceph exec -ti rook-ceph-tools ceph status # Run once to print output
```

Comment:
Does this work?  Or do you need to provide the actual name of the pod?

```
ryan@login:/homes/ryan/deepops$ kubectl -n rook-ceph exec -ti rook-ceph-tools ceph status
Error from server (NotFound): pods "rook-ceph-tools" not found
```

```
ryan@login:/homes/ryan/deepops$ kubectl -n rook-ceph exec -ti rook-ceph-tools-5bc8b8f97-77bnl ceph status
  cluster:
    id:     5e174c88-f3d5-4550-b4b9-ac2c85a95ad2
    health: HEALTH_OK
  ...
```

Reply:
Good catch. We had a little hiccup during QA/check-in. I just pushed the updated script that dynamically pulls the name.